packages/main/src/Link.ts

```diff
@@ -220,6 +225,43 @@ class Link extends UI5Element implements ITabbable {
 	@property({ type: Object })
 	accessibilityAttributes!: LinkAccessibilityAttributes;
 
+	/**
+	 * Defines the icon, displayed as graphical element within the component before the link's text.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * **Note:** We recommend using аn icon in the beginning or the end only. Using two icons is discouraged.
+	 *
+	 * Example:
+	 * See all the available icons within the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html).
+	 * @default ""
+	 * @since 2.0.0
+	 * @public
+	 */
+	@property()
+	icon!: string;
+
+	/**
+	 * Defines the icon, displayed as graphical element within the component after the link's text.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * **Note:** We recommend using аn icon in the beginning or the end only. Using two icons is discouraged.
+	 *
+	 * Example:
+	 * See all the available icons within the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html).
+	 * @default ""
+	 * @since 2.0.0
+	 * @public
+	 */
+	@property()
+	endIcon!: string;
+
+	/**
+	 * Indicates if the elements has an end icon
+	 * @private
+	 */
+	@property({ type: Boolean })
+	hasEndIcon!: boolean;
```

Comment:
why do we need that property? If it's only for CSS styling we can depend on `end-icon` attribute.

Reply:
`endIcon` is a slot, not a property (you are thinking about the previous pattern with `icon-end` boolean), so it won't bi present

Reply:
On a second look, these are properties (I thought they are slots for a moment), so `end-icon` attribute as Nayden said is present, and you can use it instead of hasEndIcon